kodilla-testing/src/main/java/com/kodilla/testing/library/BookLibrary.java

```diff
@@ -13,12 +13,23 @@ public BookLibrary(LibraryDatabase libraryDatabase) {
 
     public List<Book> listBooksWithCondition(String titleFragment) {
 
-        List<Book> bookList = new ArrayList<Book>();
+        List<Book> bookList = new ArrayList<>();
         if (titleFragment.length() < 3) return bookList;
 
         List<Book> resultList = libraryDatabase.listBooksWithCondition(titleFragment);
         if (resultList.size() > 20) return bookList;
         bookList = resultList;
         return bookList;
     }
+
+    //1 gdy użytkownik nie ma wypożyczonych żadnych książek,
```

Comment:
Co to za komentarz i dlaczego w tej klasie?

Reply:
ten komentarz jest treścią zadania; jest dla mnie. Rozumiem, że masz na myśli, że jeśli zostawiam komentarz, to też mam pamiętać o tym, że ktoś inny może go przeczytać i w tym przypadku on powinien być zrozumiły dla wszystkich, a nie tylko dla mnie? Ok :)